### PR TITLE
Add automatic cert renewal systemd service.

### PR DESCRIPTION
**Pull request by mrstegeman**

### CODE REVIEW

File: tools/renew-certificates.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+gateway_dir="/home/pi/mozilla-iot/gateway"
+
+certbot renew \
+    --config-dir "$gateway_dir/etc" \
```
Comment (hobinjk) [inline tools/renew-certificates.sh]: Is there configuration that we need to specify?

File: tools/renew-certificates.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    --logs-dir "$gateway_dir/var/log" \
```
Comment (hobinjk) [inline tools/renew-certificates.sh]: I'd prefer to not top-level `etc` and `var` directories into the gateway if we don't have to. Alternatively, they could be added to .gitignore

**Comment by mrstegeman** [comment]
I'm thinking this may not actually work, since we're using DNS challenge... I need to find a way to test renewal.

File: tools/renew-certificates.sh (hunk 1 above)
Comment (mrstegeman) [reply tools/renew-certificates.sh]: certbot will try to auto-renew any certificates it finds in the given config directory. When we generate certs, those are now written out to the filesystem, including the renewal config. See #363.

**Comment by mrstegeman** [comment]
Do not merge yet, still working through issues.

**Pull request merged**